packages/scene-composer/src/utils/entityModelUtils/modelRefComponent.spec.ts

```diff
@@ -0,0 +1,138 @@
+import { componentTypeToId } from '../../common/entityModelConstants';
+import { KnownComponentType } from '../../interfaces';
+
+import { createModelRefComponent, parseModelRefComp, updateModelRefComponent } from './modelRefComponent';
+
+describe('createModelRefComponent', () => {
+  it('should return expected modelRef component with uri and modelType', () => {
+    const result = createModelRefComponent({ type: KnownComponentType.ModelRef, uri: 'abc', modelType: 'modelType' });
+
+    expect(result.properties).toEqual({
+      uri: {
+        value: { stringValue: 'abc' },
+      },
+      modelType: {
+        value: { stringValue: 'modelType' },
+      },
+    });
+  });
+
+  it('should return expected modelRef component by setting castShadow and receiveShadow to false', () => {
+    const result = createModelRefComponent({
+      type: KnownComponentType.ModelRef,
+      uri: 'abc',
+      modelType: 'modelType',
+      unitOfMeasure: 'decimeters',
+      localScale: [1, 1, 1],
+      castShadow: false,
+      receiveShadow: false,
+    });
+
+    expect(result.properties).toEqual({
```

Comment:
result should include `castShadow` and `receiveShadow` property with value false

Reply:
If save yes we should include add false result here.